tests/DocumentTest.php

```diff
@@ -87,6 +87,223 @@ public function testGetAdSection()
         $this->assertSame('ad1', $adSection->getId());
     }
 
+    /**
+     * Test for creating media file with skipping after specific time
+     */
+    public function testCreateAdSectionWithSkipAfter()
```

Comment:
this method tests `Create Ad Section With SkipAfter` so you must check that `SkipAfter` really exiosts and has vlue that you pass. But you just check thet ad section was added. Other tests has same issue